src/interpreter/index.ts

```diff
@@ -767,7 +767,7 @@ export class Interpreter implements Expr.Visitor<BrsType>, Stmt.Visitor<BrsType>
                 let messageParts = [];
 
                 let args = sig.args.map(a => {
-                    let requiredArg = `${a.name} as ${ValueKind.toString(a.type.kind)}`;
+                    let requiredArg = `${a.name.text} as ${ValueKind.toString(a.type.kind)}`;
```

Comment:
I couldn't figure out how to test this one, but pretty sure it needed changed as well.

Reply:
IIRC this comes into play when calling something like `UCase(5)` - any type mismatch causes the function signature to be printed out before the mismatched arguments.  But it definitely needed to be fixed.  Great catch!